packages/blocks/video/src/components/Media.tsx

```diff
@@ -0,0 +1,366 @@
+import {
```

Comment:
Can we write somewhere in this file and other duplicated files that these are duplicated and `@todo` to share them? and advise people that changes in one place need to be made in the other until we share them

Reply:
👍